repos/fdbt-site/src/pages/returnService.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { ReactElement } from 'react';
+import upperFirst from 'lodash/upperFirst';
+import { ErrorInfo, NextPageContextWithSession, ServiceType, TxcSourceAttribute } from '../interfaces';
+import FormElementWrapper from '../components/FormElementWrapper';
+import TwoThirdsLayout from '../layout/Layout';
+import {
+    OPERATOR_ATTRIBUTE,
+    SERVICE_ATTRIBUTE,
+    PASSENGER_TYPE_ATTRIBUTE,
+    TXC_SOURCE_ATTRIBUTE,
+    RETURN_SERVICE_ATTRIBUTE,
+} from '../constants/attributes';
+import { getServiceByIdAndDataSource, getServicesByNocCodeAndDataSource } from '../data/auroradb';
+import ErrorSummary from '../components/ErrorSummary';
+import { getAndValidateNoc, getCsrfToken } from '../utils';
+import CsrfForm from '../components/CsrfForm';
+import { isPassengerType, isServiceAttributeWithErrors } from '../interfaces/typeGuards';
+import { getSessionAttribute, getRequiredSessionAttribute } from '../utils/sessions';
+import { redirectTo } from '../utils/apiUtils';
+
+const title = 'Return Service - Create Fares Data Service';
+const description = 'Service selection page of the Create Fares Data Service';
+const errorId = 'returnService';
+
+interface ReturnServiceProps {
+    operator: string;
+    passengerType: string;
+    services: ServiceType[];
+    error: ErrorInfo[];
+    dataSourceAttribute: TxcSourceAttribute;
+    csrfToken: string;
+    selectedServiceId: number;
+}
+
+const ReturnService = ({
+    operator,
+    passengerType,
+    services,
+    dataSourceAttribute,
+    error,
+    csrfToken,
+    selectedServiceId,
+}: ReturnServiceProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description} errors={error}>
+        <CsrfForm action="/api/returnService" method="post" csrfToken={csrfToken}>
+            <>
+                <label htmlFor="service">
+                    <h1 className="govuk-heading-l" id="service-page-heading">
+                        Select a service
```

Comment:
This page needs to have a different title and content to /service. It needs to explain to the user that they are selecting a service which is ONLY for the return leg of their return ticket, and that they are doing this because their service only has 1 direction.

This page content needs to be shown to Steve before we merge it in.

Reply:
changed the text